luigi/scheduler.py

```diff
@@ -822,7 +921,22 @@ def get_work(self, host=None, assistant=False, current_tasks=None, worker=None,
                  'task_id': None,
                  'n_unique_pending': n_unique_pending}
 
-        if best_task:
+        if len(batched_tasks) > 1:
+            batch_id = hashlib.md5('|'.join(task.id for task in batched_tasks)).hexdigest()
+            for task in batched_tasks:
+                self._state.set_batch_running(task, batch_id, worker_id)
+
+            combined_params = best_task.params.copy()
+            combined_params.update(batched_params)
```

Comment:
This is the line that made me shiver seeing `batched_params` is ever set to `None`